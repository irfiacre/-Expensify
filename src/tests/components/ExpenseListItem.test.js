import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/expenseListItem';
import expensesTestData from '../fixtures/expensesData';

test('Should render Expense List Item', ()=>{
    const wrapper = shallow( < ExpenseListItem  { ...expensesTestData[0] } /> );
    expect(wrapper).toMatchSnapshot();
});
