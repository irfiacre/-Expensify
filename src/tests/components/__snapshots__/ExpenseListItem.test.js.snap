// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Expense List Item 1`] = `
<Link
  className="listItem"
  to="/edit/1"
>
  <div>
    <h3
      className="listItemTitle"
    >
      Gum Expense
    </h3>
    <span
      className="listItemSubtitle"
    >
      January 1st, 1970
    </span>
  </div>
  <h3
    className="listItemData"
  >
    $1.95
  </h3>
</Link>
`;
